Makefile

```diff
@@ -49,3 +49,7 @@ clean:
 	$(RM) *.a *.~ *.exe *.o
 	$(MAKE) -C libmincrypt clean
 
+install:
+	install -m 755 unpackbootimg ${PREFIX}/bin
```

Comment:
Not sure why you used curly brackets instead of round where the rest of the file uses round. Again, stylistic consistency is important when contributing to a project.

Also, as I previously mentioned, I would organizationally prefer install to come before clean in the Makefile, if you could please adjust the placement.